rice-framework/krad-it/src/test/java/org/kuali/rice/krad/service/DateTimeServiceTest.java

```diff
@@ -340,4 +340,28 @@ private static void waitForMidnightIfWithinOneSecond() throws InterruptedExcepti
 
         assertEquals("-5", Integer.toString(CoreApiServiceLocator.getDateTimeService().dateDiff(date1, date2, false)));
     }
+
+    @Test public void testDateDiffDaylightSavingsTimeInclusive() throws ParseException {
```

Comment:
Can you include some commentary in this test that mentions how these tests are interacting with the DST offset? I remember now (in March 2017) that DST began a couple weeks ago, but I'd like to have that context in the future when I wonder why these dates were chosen for this test.